contrib/test_jenkins.sh

```diff
@@ -662,73 +701,67 @@ run_uct_hello() {
 }
 
 run_client_server() {
+	test_name=ucp_client_server
 
-    test_name=ucp_client_server
-
-    if [ ! -x ${test_name} ]
-    then
-        gcc -o ${test_name} ${ucx_inst}/share/ucx/examples/${test_name}.c \
-            -lucp -lucs -I${ucx_inst}/include -L${ucx_inst}/lib \
-            -Wl,-rpath=${ucx_inst}/lib
-    fi
-
-    iface=`ibdev2netdev | grep Up | awk '{print $5}' | head -1`
-    if [ -n "$iface" ]
-    then
-        server_ip=`ip addr show ${iface} | awk '/inet /{print $2}' | awk -F '/' '{print $1}'`
-    fi
-
-    if [ -z "$server_ip" ]
-    then
-        # if there is no inet (IPv4) address, bail
-        return
-    fi
-
-    ibdev=`ibdev2netdev | grep $iface | awk '{print $1}'`
-    node_guid=`cat /sys/class/infiniband/$ibdev/node_guid`
-    if [ $node_guid == "0000:0000:0000:0000" ]
-    then
-        return
-    fi
+	if [ ! -x ${test_name} ]
+	then
+		gcc -o ${test_name} ${ucx_inst}/share/ucx/examples/${test_name}.c \
+			-lucp -lucs -I${ucx_inst}/include -L${ucx_inst}/lib \
+			-Wl,-rpath=${ucx_inst}/lib
+	fi
 
-    server_port=$((10000 + EXECUTOR_NUMBER))
+	iface=`ibdev2netdev | grep Up | awk '{print $5}' | head -1`
+	if [ -n "$iface" ]
+	then
+		server_ip=`ip addr show ${iface} | awk '/inet /{print $2}' | awk -F '/' '{print $1}'`
+	fi
 
-    # run server side
-    ./${test_name} -p ${server_port} &
-    hw_server_pid=$!
+	if [ -z "$server_ip" ]
+	then
+		# if there is no inet (IPv4) address, bail
+		return
+	fi
 
-    sleep 15
+	ibdev=`ibdev2netdev | grep $iface | awk '{print $1}'`
+	node_guid=`cat /sys/class/infiniband/$ibdev/node_guid`
+	if [ $node_guid == "0000:0000:0000:0000" ]
+	then
+		return
+	fi
 
-    # need to be ran in background to reflect application PID in $!
-    ./${test_name} -a ${server_ip} -p ${server_port} &
-    hw_client_pid=$!
+	server_port=$((10000 + EXECUTOR_NUMBER))
```

Comment:
i think run_client_server_app() function can calculate server_port inside, no need to pass it as param